integration_tests/dockerfiles/Dockerfile_test_add

```diff
@@ -15,4 +15,4 @@ ADD context/tars/fil* /tars/
 ADD context/tars/file.tar /tars_again
 
 # Finally, test adding a remote URL, concurrently with a normal file
-ADD https://github.com/GoogleCloudPlatform/docker-credential-gcr/releases/download/v1.4.3/docker-credential-gcr_linux_386-1.4.3.tar.gz context/foo /test/all/
+ADD https://github.com/GoogleContainerTools/docker-credential-gcr/releases/download/v1.4.3/docker-credential-gcr_linux_386-1.4.3.tar.gz context/foo /test/all/
```

Comment:
Looks like [docker-credential-gcr](https://github.com/GoogleCloudPlatform/docker-credential-gcr) is still in GoogleCloudPlatform

Reply:
oops